src/lib/config.js

```diff
@@ -19,7 +19,6 @@ config.sqlInitFiles = [
     'section',
     'student',
     'isOwner',
-    'isCourseInstance',
```

Comment:
If we're going to be removing this, should also add the necessary config to the `course_instances` schema

Reply:
Thought you already added a course key for some random reason.